src/styles/main.scss

```diff
@@ -62,6 +66,7 @@
 // The secondary column is hidden on small screens
 .algolia-docsearch-suggestion--subcategory-column {
   display: none;
+  cursor: default;
```

Comment:
Not sure it's such a good idea to remove the `cursor: pointer` on something clickable.